.env.dusk.ci

```diff
@@ -43,6 +43,7 @@ NORTHSTAR_AUTH_ID=
 NORTHSTAR_AUTH_SECRET=
 NORTHSTAR_CLIENT_SECRET=
 NORTHSTAR_CLIENT_ID=
+NORTHSTAR_EMAIL=
```

Comment:
I think you might need to leave these lines commented out for the Wercker values to be used.